src/components/Users/UserProfile.tsx

```diff
@@ -494,8 +494,6 @@ export default function UserProfile() {
           refetchUser();
           Notification.Success({ msg: "Profile picture updated." });
           setEditAvatar(false);
-        } else {
-          onError();
```

Comment:
Why is this removed? 🤔

Reply:
yes sir, actually this else was just added 20hrs ago, it shows in my git history
![image](https://github.com/user-attachments/assets/e327282f-5787-43f4-a4ef-79ba2888ba30)

so I was getting errors while generating PR during git commit, so thats why the lines got removed unknowingly as it seemed up to date

but i will pull from develop and update the PR

Reply:
my branch is 3 commits behind the history says, it was not in the morning ig! I will update

Reply:
issue solved sir, merging and pulling was not working, so manually added else block, sry this time

Reply:
Hmm, ok.

Note: As Bodhish mentioned before, you can just refer to us by our names 👍